Resources/public/js/alloyeditor/toolbars/config/block-base.js

```diff
@@ -16,10 +16,12 @@ YUI.add('ez-alloyeditor-toolbar-config-block-base', function (Y) {
     var ReactDOM = Y.eZ.AlloyEditor.ReactDOM;
 
     function outlineTotalWidth(block) {
-        return (
-            parseInt(block.getComputedStyle('outline-offset'), 10) +
-            parseInt(block.getComputedStyle('outline-width'), 10)
-        );
+        var outlineOffset = parseInt(block.getComputedStyle('outline-offset'), 10);
+        var outlineWidth = parseInt(block.getComputedStyle('outline-width'), 10);
+        if ( isNaN(outlineOffset) ) {
+            outlineOffset = 1;
```

Comment:
oh yeah because of https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/css/theme/alloyeditor/content.css#L9

this deserves some comments in both place to clarify a bit that. Here, something like:

```js
if ( isNaN(outlineOffset) ) {
    // Edge does not support offset-offset yet
    // 1 comes from the stylesheet, see theme/alloyeditor/content.css
    outlineOffset = 1;
}
```

and in the corresponding CSS, I would add something like:

```
.ez-richtext-editable .is-block-focused,
.ez-richtext-editable .cke_widget_wrapper.cke_widget_focused>.cke_widget_element {
    outline: 2px dashed #aaa;
    /* when changing outline-offset remember to also change
     * the outline width computation for Edge in
     * js/alloyeditor/toolbars/config/block-base.js
     */
    outline-offset: 1px;
}
```